### PR TITLE
Add first unit test and CI job

### PR DESCRIPTION
**Pull request by EmanueleGallone**
PR's aim is to provide an initial unit test for far_builder and add a CI job executing unit tests for `pfcpsim`

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -20,3 +20,10 @@ build-pfcpsim-client:
 
 golint:
 	@docker run --rm -v $(CURDIR):/app -w /app/pkg/pfcpsim golangci/golangci-lint:latest golangci-lint run -v --config /app/.golangci.yml
+
+.coverage:
+	rm -rf $(CURDIR)/.coverage
+	mkdir -p $(CURDIR)/.coverage
+
+test: .coverage
+	go test	-race -coverprofile=.coverage/coverage-unit.txt -covermode=atomic -v ./...
```
Comment (EmanueleGallone) [inline Makefile:29]: Is it ok to run directly `go test` ? Let me know if you think that we should run test in a container

File: pkg/pfcpsim/session/far_builder_test.go [hunk 2]
```diff
@@ -0,0 +1,34 @@
+package session
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestFARBuilderShouldPanic(t *testing.T) {
+	assert.Panics(t, func() {
```
Comment (EmanueleGallone) [inline pkg/pfcpsim/session/far_builder_test.go:10]: Didn't find a proper way to setup a table testing scenario while testing panics

Comment (EmanueleGallone) [reply pkg/pfcpsim/session/far_builder_test.go:10]: maybe I could do something like this: https://gist.github.com/Nitecon/3e10e05af19b13f21e77a8160f138c58 But I don't like it much

File: Makefile (hunk 1 above)
Comment (osinstom) [reply Makefile:29]: It's ok for me

**Review by osinstom — APPROVED** [review]

**Pull request merged**